client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -323,12 +328,81 @@ def predict_with_id(
             max_retries=max_retries,
             status_forcelist=retry_status,
             backoff_factor=backoff_factor,
-            )
+        )
 
-        response = self._predict(x, compress, prediction_id)
+        response = self._predict(x, self.prediction_url, compress, prediction_id)
         id = response.headers.get('verta-request-id', '')
         return (id, _utils.body_to_json(response))
 
+    def batch_predict(
+            self,
+            df,
```

Comment:
Ahhhhh we can't use `pandas` in type annotations here, huh 😞

Reply:
sadly not :(